### PR TITLE
 docs: updating governance

### PR DESCRIPTION
**Pull request by alyssawilk**

### CODE REVIEW

**Comment by alyssawilk** [comment]
@envoyproxy/envoy-maintainers for review

File: GOVERNANCE.md [hunk 1]
```diff
@@ -1,34 +1,22 @@
...
-  organization.
+* Express interest to the
+  [envoy-maintainers](https://groups.google.com/forum/#!forum/envoy-announce)
+  that you are interested in becoming a maintainer and, if your company does not have pre-existing maintainers,
+  that your organization is interested in and willing to sponsoring a maintainer.
 * We will expect you to start contributing increasingly complicated PRs, under the guidance
-  of the existing senior maintainers.
-* We may ask you to do some PRs from our backlog.
+  of the existing maintainers.
+* We may ask you to do some issues from our backlog.
```
Comment (RyanTheOptimist) [inline GOVERNANCE.md:12]: nit: "do some issues" is a bit awkward. Perhaps something like "fix some issues" or "work on some issues". WDYT?

**Review by mattklein123 — APPROVED** [review]

**Review by jmarantz — APPROVED** [review]

**Review by ggreenway — APPROVED** [review]

**Review by phlax — APPROVED** [review]
lgtm, thanks @alyssawilk

**Review by adisuissa — APPROVED** [review]
LGTM.

**Review by tyxia — APPROVED** [review]
LGTM, thanks

**Pull request merged**